lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -35,9 +35,7 @@ class EstimatedInputLatency extends Audit {
       name: 'estimated-input-latency',
       description: 'Estimated Input Latency',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
-      helpText: 'Primarily a measure of the main thread\'s activity. ' +
-      'There is a 90% probabililty a user would encounter input latency of this amount or less. ' +
-      '10% of touches can expect additional latency.',
+      helpText: 'Input latency is a key factor in how users perceive your app\'s performance. If your app takes more than 100ms to respond to user input, users perceive the app as laggy. <a href="https://developers.google.com/web/tools/lighthouse/audits/estimated-input-latency" rel="noopener noreferrer" target="_blank">Learn more</a>.',
```

Comment:
I think we need some part of the previous `helpText` here, otherwise this number has no context. It's specifically the input latency at the 90th percentile.

A touch on the screen has some number of milliseconds before the page responds due to main thread activity getting in the way. This audit is saying that for the window we looked at, 90% of the time we'd expect that latency to be this number or less, with the goal being that 90% of the time the page should respond within 50ms or less.